### PR TITLE
[DOCU-2314] Konnect Service Hub: UI validation and style cleanup

### PR DESCRIPTION
**Pull request by lena-larionova**
### Summary
* Adding section on labels to the service doc
* Updating character requirements for service display names and gateway service names
* Updating incompatible plugins list
* Capitalization, phrasing, consistency cleanup
* Short descriptions for deleting stuff to say what all is being deleted

### Reason
https://konghq.atlassian.net/browse/DOCU-2314

### Testing
Everything under https://deploy-preview-4082--kongdocs.netlify.app/konnect/servicehub/

### CODE REVIEW

File: app/konnect/servicehub/index.md [hunk 1]
```diff
@@ -42,41 +42,41 @@ group for the version to run on.
...
 listens for requests. You can specify the URL with a single string, or by
 specifying its protocol, host, port, and path individually.
 
-[Get started with Service management &rarr;](/konnect/servicehub/manage-services)
+[Get started with service management &rarr;](/konnect/servicehub/manage-services)
 
 ### {{site.base_gateway}} implementations
 
 When configuring a {{site.base_gateway}} implementation of a service, you'll
 need to specify a route. Routes determine how (and if) requests get sent to
```
Comment (Guaris) [inline app/konnect/servicehub/index.md:54]:
```suggestion
need to specify a route. Routes determine how successful requests are sent to
```

File: app/konnect/servicehub/manage-services.md [hunk 2]
```diff
@@ -10,55 +10,74 @@ implementations in one place.
...
-1. In the {% konnect_icon servicehub %} Service Hub, click **+ New service**.
+1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), click the **New service** button.
 
-1. Enter a **Display name**.
+1. Enter a display name.
 
-    A display name can be any string containing letters, numbers, or the following
-    characters: `.`, `-`, `_`, `~`, or `:`. Do not use spaces in display names.
+    A display name can be any string containing letters, numbers, spaces, or the following
+    characters: `.`, `-`, `_`, `~`, or `:`. Spaces are equal to the `-` character.
```
Comment (Guaris) [inline app/konnect/servicehub/manage-services.md:20]:
```suggestion
    characters: `.`, `-`, `_`, `~`, `:`. Spaces are equal to the `-` character.
```

File: app/konnect/servicehub/manage-services.md [hunk 3]
```diff
@@ -10,55 +10,74 @@ implementations in one place.
...
+1. In the header next to **Labels**, click **Edit**.
+
+1. Click **Add label** and add any labels in `key:value` pair format.
 
-## Update a Service
+    For example, you might set `location:us-west`, where `location` is the key
+    and the `us-west` is the value.
 
-1. In the {% konnect_icon servicehub %} Service Hub, select a Service from the list.
+    These labels are completely custom. Set anything that you need.
```
Comment (Guaris) [inline app/konnect/servicehub/manage-services.md:43]:
```suggestion
    These labels are customizable, you can set them to any value.
```

File: app/konnect/servicehub/plugins.md [hunk 4]
```diff
@@ -25,7 +27,7 @@ through the Runtime Manager.
 routes, and consumers in a runtime group &ndash; set it up through the
```
Comment (Guaris) [inline app/konnect/servicehub/plugins.md:27]:
```suggestion
routes, and consumers in a runtime group, using the
```

File: app/konnect/servicehub/service-documentation.md [hunk 5]
```diff
@@ -3,108 +3,94 @@ title: Dev Portal Service Documentation
 no_version: true
 ---
 
-A core function of the Dev Portal is publishing Service descriptions and API specs. Developers can use the spec and corresponding descriptions to access, consume, and register new applications against your Services.
-
-Through Service Hub, you can also publish any Service in your catalog and its
-documentation to the Dev Portal. Publishing services to the Dev Portal is the only way to expose your service to developers. Once the Service is published and available to developers, they can apply for access by [registering](/konnect/dev-portal/dev-reg/) a developer account. You can also [manage](/konnect/dev-portal/access-and-approval/manage-devs/) access to the Dev Portal from the {{site.konnect_saas}} interface.
+A core function of the Dev Portal is publishing service descriptions and API specs. Developers can use the spec and corresponding descriptions to access, consume, and register new applications against your services.
```
Comment (Guaris) [inline app/konnect/servicehub/service-documentation.md:6]:
```suggestion
A core function of the Dev Portal is publishing service descriptions and API specs. Developers can use the Dev Portal to access, consume, and register new applications against your services.
```
I thought the core function was to expose services and allow developers to manage their own credentials for their applications.

File: app/konnect/servicehub/service-documentation.md [hunk 6]
```diff
@@ -3,108 +3,94 @@ title: Dev Portal Service Documentation
...
 
 ## API specification
 
-API specifications, or specs, can be uploaded and attached to a specific Version within your Dev Portal. You can have different API specs that correspond to different versions. {{site.konnect_short_name}} accepts OpenAPI (Swagger) specs in YAML or JSON.
-
-### Upload a Version Spec
-
-1. From the left navigation menu, open the {% konnect_icon servicehub %} **Service Hub** page.
+API specifications, or specs, can be uploaded and attached to a specific version within your Dev Portal.
+You can have different API specs that correspond to different versions.
```
Comment (Guaris) [inline app/konnect/servicehub/service-documentation.md:46]:
```suggestion
You can have different API specs that correspond to different service versions.
```

File: app/konnect/servicehub/service-documentation.md [hunk 7]
```diff
@@ -3,108 +3,94 @@ title: Dev Portal Service Documentation
...
+1. In the {% konnect_icon servicehub %} [**Service Hub**](https://cloud.konghq.com/servicehub), select a service from the list.
 
-2. Click on the **Service actions** dropdown menu and select **Publish to portal**.
+1. Click the **Service actions** drop-down menu and select **Publish to portal**.
 
-    This publishes a Service's API specs to the Dev Portal.
+### Unpublish a service {#unpublish}
 
-### Unpublish a Service {#unpublish}
+Unpublish a service to remove it from the Dev Portal. Removing a service from the Dev Portal is not permanent, and you can republish again at any time.
```
Comment (Guaris) [inline app/konnect/servicehub/service-documentation.md:92]:
```suggestion
Unpublish a service to remove it from the Dev Portal. Removing a service from the Dev Portal is not permanent, and you can republish it at any time.
```

File: app/konnect/servicehub/service-implementations.md [hunk 8]
```diff
@@ -3,84 +3,85 @@ title: Manage Services through ServiceHub
...
 1. In the **Create Implementation** dialog, in step 1, enter the connection
 details for the upstream service.
 
-    1. Enter a name for the Gateway Service.
+    1. Enter a name for the Gateway service.
 
-        This name must be unique to the runtime group. You can't use an
-        existing Gateway Service here.
+        The name can be any string containing letters, numbers, or the following
+        characters: `.`, `-`, `_`, `~`, or `:`. Do not use spaces.
```
Comment (Guaris) [inline app/konnect/servicehub/service-implementations.md:30]:
```suggestion
        characters: `.`, `-`, `_`, `~`, `:`. Do not use spaces.
```

File: app/konnect/servicehub/service-versions.md [hunk 9]
```diff
@@ -7,64 +7,62 @@ Every {{site.konnect_short_name}} service version is associated with one [runtim
...
     [application registration](/konnect/dev-portal/applications/application-overview),
     choose `default` in this step.
 
-    Different versions of the same Service can run in different runtime groups.
+    Different versions of the same service can run in different runtime groups.
     The version name is unique within a group:
 
-    * If you create multiple versions in the **same group**, they must have unique names.
-    * If you create multiple versions in **different groups**, the versions can have the same name.
+    * If you create multiple versions in the _same group_, they must have unique names.
```
Comment (Guaris) [inline app/konnect/servicehub/service-versions.md:55]:
```suggestion
    * If you create multiple versions in the _same group_, the versions must have unique names.
```

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/konnect/servicehub/index.md [hunk 10]
```diff
@@ -13,26 +13,26 @@ entire architecture.
 
 ## Services in the Service Hub catalog
 
-Each entry in the Service Hub is called a **{{site.konnect_short_name}} service**, or **service**
+Each entry in the Service Hub is called a _{{site.konnect_short_name}} service_, or _service_
 for short. This is the abstraction of one of your own upstream services.
```
[the remark this answers is not in the document]
Comment (lena-larionova) [reply app/konnect/servicehub/index.md:17]: Adjusted your comment for typo + shortname.

File: app/konnect/servicehub/service-documentation.md (hunk 5 above)
Comment (lena-larionova) [reply app/konnect/servicehub/service-documentation.md:6]: Yep, definitely a copy-paste typo from ages ago.

**Review by Guaris — APPROVED** [review]

**Pull request merged**